src/RequestLog/Services/RequestLogOptionsService.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Cego\RequestLog\Services;
+
+use Cego\RequestLog\Models\RequestLogOption;
+use Illuminate\Support\Facades\Cache;
+
+class RequestLogOptionsService {
+    const REQUEST_LOG_ENABLED_KEY = "request_log_enabled";
+
+    /**
+     * Checks if Request Log is disabled or not, cached
+     *
+     * @return bool
+     */
+    public function isRequestLogEnabledCached(): bool
```

Comment:
Kan vi ikke lade være med at have 2 metoder til dette? Og blot lade et argument bestemme om man ønsker raw version, og lade default være cache versionen?

Reply:
Har svært ved at se hvordan det er pænere, da der så skal være en større kompleks metode fremfor to gode og simple metoder med DRY compliance?